apps/volk_option_helpers.cc

```diff
@@ -1,6 +1,25 @@
-//
-// Created by nathan on 2/1/18.
-//
+/* -*- c++ -*- */
+/*
+ * Copyright 2018 Free Software Foundation, Inc.
```

Comment:
I'd suggest changing this to 2018-2020 ... there are enough changes herein to warrant it!